Makefile

```diff
@@ -64,7 +64,7 @@ unittest: .venv
 	@echo '==> Running unit tests'
 	.venv/bin/pytest -m "not integration" tests
 
-test: check-formatting check-linting check-typing unittest version-tracker
+test: check-formatting check-linting check-typing unittest install-vscode-extensions version-tracker
```

Comment:
How about keeping this separate from `make test`? Installing extensions as part of "testing" is an unexpected side effect.

Reply:
That's a good point, but currently we (at least I) use `make test` to set up an environment from scratch. What would you suggest otherwise?

Reply:
Just to let people run both `make test` and `make install-vscode-extensions` separately. Only a few of us will be using the extension, so there's no need to install it by default. (I did the same thing with installing `uv` as part of `make test` and Lars wanted me to put it away... which I haven't done yet 😅).

Reply:
On second thought, I think it makes sense to separate the installation of the environment and the extensions (especially when thinking of external users who may just want to try the ETL for a specific thing and are not going to develop any steps). I'll create a PR.